### PR TITLE
Default to using grpc tritonclient

### PR DESCRIPTION
**Pull request by benfred**

### CODE REVIEW

File: tests/unit/test_triton_inference.py [hunk 1]
```diff
@@ -6,10 +6,9 @@
...
 import nvtabular.ops as ops
 
-# triton = pytest.importorskip("nvtabular.inference.triton")
+triton = pytest.importorskip("nvtabular.inference.triton")
 
 
-@pytest.mark.skip(reason="not playing nice with others")
 @pytest.mark.parametrize("engine", ["parquet"])
 def test_generate_triton_model(tmpdir, engine, df):
     import nvtabular.inference.triton as triton
```
Comment (jperez999) [inline tests/unit/test_triton_inference.py:14]: remove this import I added for the flake8 stuff. so triton mention below would still be tied to something... with uncommenting above we don't need to have this import here

**Review by jperez999 — CHANGES_REQUESTED** [review]

File: tests/unit/test_triton_inference.py (hunk 1 above)
Comment (benfred) [reply tests/unit/test_triton_inference.py]: done

**Review by jperez999 — APPROVED** [review]

**Comment by benfred** [comment]
rerun tests

**Comment by benfred** [comment]
The issue seems to be because ```tritonclient.http``` imports gevent - and gevent does something to mess up the JIT compile with cudf. Changing to default to grpc here causes the issue in https://github.com/NVIDIA/NVTabular/issues/616 to go away.

**Pull request merged**